release-schema.json

```
@@ -98,6 +98,45 @@
             "null"
           ],
           "minLength": 1
+        },
+        "advancedElectronicSignatureRequired": {
+          "title": "Advanced electronic signature required",
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether an advanced or qualified electronic signature or seal (as defined in European Parliament and Council Regulation (EU) No 910/2014) is required."
+        },
+        "multipleBidsAllowed": {
+          "title": "Multiple bids allowed",
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether tenderers can submit more than one bid."
+        },
+        "nonElectronicRationale": {
```

The field this relates to is `electronicSubmissionPolicy`. Let's make that clearer.

Need to update the example and changelog.

```suggestion
        "nonElectronicSubmissionRationale": {
```

---

Example and changelog updated in https://github.com/open-contracting-extensions/ocds_submissionTerms_extension/pull/4/commits/8f7fe4c302dc7296b5714183b741399bffa14cd9